test/algorithms/overlay/overlay.cpp

```diff
@@ -368,7 +370,6 @@ void test_all()
 
 
     TEST_UNION(case_134_multi, 66.0, 1, 2);
-    return;
```

Comment:
Note, most of these tests are repeated in other unit-tests and for my own investigations.
Earlier they were not there - and maybe we could remove them.
For now, I just fixed the test and removed the `return;` which is not intended to be commited.